modules/candidate_parameters/ajax/getData.php

```diff
@@ -15,7 +15,9 @@
 use \LORIS\StudyEntities\Candidate\CandID;
 
 $user = \NDB_Factory::singleton()->user();
-if (!$user->hasPermission('candidate_parameter_edit')) {
+if (!$user->hasPermission('candidate_parameter_edit')
+    && !$user->hasPermission('candidate_parameter_view')
```

Comment:
You could also use the following fucntion:

```
if (!$user->hasAnyPermission(
      array(
             'candidate_parameter_edit',
             'candidate_parameter_view'
      )
)) { ...
```

Reply:
Mmmm...Didn't know about that one! Thanks!